src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -348,12 +396,7 @@ const ArtworkSidebarCommerialButtons: React.FC<ArtworkSidebarCommercialButtonsPr
 
       <Flex flexDirection={["column", "column", "column", "column", "row"]}>
         <Join separator={<Spacer x={1} y={1} />}>
-          {!!isCreateAlertAvailable && (
-            <ArtworkCreateAlertButtonFragmentContainer
-              artwork={artwork}
-              analyticsContextModule={ContextModule.artworkSidebar}
-            />
-          )}
+          <ArtworkAlertSwitch />
```

Comment:
**walkthrough**: Between the `isCreateAlertAvailable` and `newAlertModalEnabled` conditions, it was difficult to inline the flow, so a switch component is rendered instead.